webapp/components/wpt-metadata.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright 2018 The WPT Dashboard Project. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+import '../node_modules/@polymer/iron-collapse/iron-collapse.js';
+import '../node_modules/@polymer/paper-button/paper-button.js';
+import '../node_modules/@polymer/polymer/lib/elements/dom-if.js';
+import '../node_modules/@polymer/polymer/lib/elements/dom-repeat.js';
+import {
+  html,
+  PolymerElement
+} from '../node_modules/@polymer/polymer/polymer-element.js';
+
+class WPTMetadataNode extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        .metadataNode {
+          display: flex;
+          align-items: center;
+          margin-bottom: 4px;
+        }
+        .metadataNode img {
+          margin-right: 16px;
+          height: 24px;
+          width: 24px;
+        }
+      </style>
+      <div class="metadataNode">
+        <iron-icon icon="bug-report"></iron-icon>
+        <div>
+          [[metadataNode.test]] :
+          <a href="[[href]]">[[metadataNode.url]]</a>
+          <br />
+        </div>
+      </div>
+    `;
+  }
+
+  static get is() {
+    return 'wpt-metadata-node';
+  }
+
+  static get properties() {
+    return {
+      metadataNode: Object,
+      href: {
+        type: String,
+        computed: 'computeHref(metadataNode)'
+      }
+    };
+  }
+
+  computeHref(metadataNode) {
+    const prefix = 'https://';
+    if (metadataNode.url.substr(0, prefix.length) !== prefix) {
+      return prefix + metadataNode.url;
+    }
+    return metadataNode.url;
+  }
+}
+window.customElements.define(WPTMetadataNode.is, WPTMetadataNode);
+
+class WPTMetadata extends PolymerElement {
+  static get template() {
+    return html`
+      <style>
+        h4 {
+          margin-bottom: 0.5em;
+        }
+      </style>
+      <template is="dom-if" if="[[firstThree]]">
+        <h4>Triaged Metadata in <i>[[path]]</i></h4>
+      </template>
+      <template is="dom-repeat" items="[[firstThree]]" as="metadataNode">
+        <wpt-metadata-node metadata-node="[[metadataNode]]"></wpt-metadata-node>
+      </template>
+      <template is="dom-if" if="[[others]]">
+        <iron-collapse id="metadata-collapsible">
+          <template is="dom-repeat" items="[[others]]" as="metadataNode">
+            <wpt-metadata-node
+              metadata-node="[[metadataNode]]"
+            ></wpt-metadata-node>
+          </template>
+        </iron-collapse>
+        <paper-button id="metadata-toggle" onclick="[[openCollapsible]]"
+          >Show more</paper-button
+        >
+      </template>
+      <br />
```

Comment:
```suggestion
      <br>
```
XHTML requires tags to be closed. HTML does not.